### DISCUSSION TITLE
Collapsible UI (Tabs/Top Bars Especially)

### DISCUSSION THREAD

**Discussion by Foxxsayy** [Floorp-Projects/Floorp#926] [Feature Request]
I just switched to Floorp and I love it! Thanks for making it! I'm still learning, so please tell me if this exists already.

I'd love to be able to totally hide the UI elements until mousing over them, like the bookmarks bar does. I'd like it for any given element (sidebars, top bars, etc), but especially horizontal tabs. For instance, Multi-Row Tab Bar looks great, but it takes up a lot of screen space. I'd like to be able to hide the tab bars or x lines of the tab bar.

(*Vertical tabs are okay, I think I'd use them if I could put them full-size on the right-hand side and toggle the open panel on/off by co-opting the Caps-Lock key or something. Or maybe a shortcut that opens the tab bar, makes the mouse scroll through tabs, and switches to the highlighted tab when released or on the next mouse click. That would actually be really useful.)

**Comment by CutterKnife** [comment]
As for the sidebar, it may be implemented in the future. Someone sent a pull request.
A setting to automatically hide the tab bar does not exist as far as I can remember, but I believe it is possible using CSS.
For example, [this theme](https://github.com/Shina-SG/Shina-Fox) has a close implementation.

**Reply by Foxxsayy** [reply]
Thanks! I'll look into it.
